strider/trapi.py

```diff
@@ -492,6 +498,16 @@ def convert_subclasses_to_aux_graphs(
                             )
                         )
 
+                    message.knowledge_graph.edges[primary_edge_id].attributes.add(
+                        Attribute.parse_obj(
+                            {
+                                "attribute_type_id": "biolink:knowledge_level",
+                                "value": "logical_entailment",
+                                "attribute_source": "infores:aragorn",
+                            }
+                        )
+                    )
+
```

Comment:
We need to also remove the existing `knowledge_level` attribute from this edge that would have been created by the KP.

Reply:
Now we'll overwrite knowledge_level if it's present, otherwise add a new attribute